js/modules/k6/http/request.go

```diff
@@ -520,13 +528,8 @@ func (h *HTTP) parseBatchRequest(
 		if p, ok := data["params"]; ok {
 			params = rt.ToValue(p)
 		}
-
-	default:
-		// Handling of "http://example.com/" or http.url`http://example.com/{$id}`
-		reqURL, err = ToURL(data)
-		if err != nil {
-			return nil, err
-		}
+	case string, httpext.URL:
```

Comment:
Can you keep the comment from the previous `default` clause ?

Reply:
Hhmm but the comment doesn't make sense anymore since it was for the `ToURL()` call, which is now done in `parseRequest()`.

Though would it be better to leave this as `default` and just pass anything to `rt.ToValue()`?